src/activity_handlers.cpp

```diff
@@ -1821,23 +1821,11 @@ void activity_handlers::pickaxe_finish( player_activity *act, player *p )
         const int helpersize = get_player_character().get_num_crafting_helpers( 3 );
         if( here.is_bashable( pos ) && here.has_flag( flag_SUPPORTS_ROOF, pos ) &&
             here.ter( pos ) != t_tree ) {
-            // Tunneling through solid rock is hungry, sweaty, tiring, backbreaking work
-            // Betcha wish you'd opted for the J-Hammer ;P
-            p->mod_stored_nutr( 15 - ( helpersize * 3 ) );
-            p->mod_thirst( 15 - ( helpersize * 3 ) );
-            if( p->has_trait( trait_STOCKY_TROGLO ) ) {
-                // Yep, dwarves can dig longer before tiring
-                p->mod_fatigue( 20 - ( helpersize  * 3 ) );
-            } else {
-                p->mod_fatigue( 30 - ( helpersize  * 3 ) );
-            }
+            // Tunneling through solid rock is sweaty, backbreaking work
+            // Betcha wish you'd opted for the J-Hammer
             p->mod_pain( std::max( 0, ( 2 * static_cast<int>( rng( 1, 3 ) ) ) - helpersize ) );
```

Comment:
```suggestion
            if( p->has_trait( trait_STOCKY_TROGLO ) ) {
                p->mod_pain( std::max( 0, ( 1 * static_cast<int>( rng( 0, 3 ) ) ) - helpersize ) );
            } else {
                p->mod_pain( std::max( 0, ( 2 * static_cast<int>( rng( 1, 3 ) ) ) - helpersize ) );
            }
```
I am suggesting this as a matter of game balance - there is little enough already making the Troglodyte mutation path attractive, and they are supposed to function better underground.

Reply:
BTW, ping: @anothersimulacrum - your thoughts?

Reply:
I don't have any opinion on this.

Reply:
Should I commit this suggestion, then? It doesn't really belong in the "Bugfixes" category of this PR, more like "Balance" (if that matters haha)

Reply:
When I get a chance, or if you want to do it, I'll check the git "blame" for who originally put in the conditional, and see about "pinging" them for their opinion. In the meantime, you might put "[CR]" (Comments Requested) at the start of the PR's title (you can modify it at the top of the PR, with the edit button beside the title).

Reply:
Ping: @John-Candlebury, @I-am-Erk, @KorGgenT? (Going off of #38505 - original account deleted, unfortunately, but these were those that were asked to review.)

Reply:
I just checked the history on STOCKY_TROGLO as a mutation. Ping @KA101, who created it; the tagline for [the commit](https://github.com/CleverRaven/Cataclysm-DDA/commit/05d92e3bc412826d2c9a94a97b29447bf9254137) adding STOCKY_TROGLO is "Get digging!".

Reply:
> Should I commit this suggestion, then? It doesn't really belong in the "Bugfixes" category of this PR, more like "Balance" (if that matters haha)

You're preserving some behaviour of STOCK_TROGLO being better at digging, I think it's fine to do here.